homeassistant/components/moscow_yandex_transport/sensor.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+"""
+Service for obtaining information about closer bus from Transport Yandex Service
+"""
+
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+from moscow_yandex_transport import YandexMapsRequester
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME, ATTR_ATTRIBUTION
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+STOP_NAME = "Stop name"
```

Comment:
Please name constants that are part of state attributes `ATTR_*`. Rename this to `ATTR_STOP_NAME`.

State attribute keys should be lowercase snakecase.

```py
ATTR_STOP_NAME = "stop_name"
```